cypress/integration/contact/create-contact.spec.js

```diff
@@ -0,0 +1,56 @@
+/// <reference types='Cypress' />
+
+const endPoint = require('../../fixtures/endpoint/endpoint.json')
+const {pageTransporter} = require('../../src/salesforce/transporter')
+const contact = require('../../src/salesforce/contact/contact')
+const newContact = require('../../src/salesforce/contact/new-contact')
+const detailContact = require('../../src/salesforce/contact/detail-contact')
+const data = require('../../fixtures/features/contact.json')
+
+
```

Comment:
There should only be one space and not two

Reply:
i already solved